data/data-pipeline/data_pipeline/tests/sources/national_risk_index/data/output.csv

```diff
@@ -1,6 +1,56 @@
 GEOID10_TRACT,FEMA Risk Index Expected Annual Loss Score,Expected population loss rate (Natural Hazards Risk Index),Expected agricultural loss rate (Natural Hazards Risk Index),Expected building loss rate (Natural Hazards Risk Index)

```

Comment:
Is this what the ETL outputs now? I'm sorry to be super nitpicky but I'm wondering if we can write a lot more digits here (like 10? idk) because the `Expected population loss rate` is so low that all the readings are clumped together at 6 significant digits.